op-e2e/actions/garbage_channel_out.go

```diff
@@ -159,9 +159,12 @@ func (co *GarbageChannelOut) AddBlock(rollupCfg *rollup.Config, block *types.Blo
 		buf.Reset()
 		buf.Write(bufBytes)
 	}
-	if co.rlpLength+buf.Len() > rollup.SafeMaxRLPBytesPerChannel {
+
+	var chainSpec = rollup.NewChainSpec(rollupCfg)
+	var maxRLPBytesPerChannel = chainSpec.MaxRLPBytesPerChannel(block.Time())
```

Comment:
```suggestion
	chainSpec := rollup.NewChainSpec(rollupCfg)
	maxRLPBytesPerChannel := chainSpec.MaxRLPBytesPerChannel(block.Time())
```
we prefer to use the `:=` syntax to define a new variable with a given value, which is more common.

Reply:
thanks, fixed